tests/test_views.py

```diff
@@ -64,7 +65,7 @@
 )
 from website.settings import ALL_MY_REGISTRATIONS_ID, ALL_MY_PROJECTS_ID
 
-from tests.base import DEFAULT_METASCHEMA
+DEFAULT_METASCHEMA = get_default_metaschema()
```

Comment:
Ok, you get the idea. 

If it's broken I'll stop asking about this line and just assume you'll do a search for all the usages. If it works as intended, great.